app/access_policies/application_user_access_policy.rb

```diff
@@ -5,8 +5,8 @@ def self.action_allowed?(action, requestor, application_user)
     # Human users are not allowed
     return false if requestor.is_human?
 
-    # Apps can only call search, updates and updated
-    return [:search, :updates, :updated].include?(action)
+    # Apps can only call read, search, updates and updated
+    return [:read, :search, :updates, :updated].include?(action)
```

Comment:
Applications should only allowed to read their own `ApplicationUser`s, so this access policy needs to enforce that requirement.  Means `:read` should be checking that the requesting application actually owns the `application_user`.